docs/user/tutorial/quickstart/java11.rst

```diff
@@ -0,0 +1,143 @@
+
+   * JAVA_HOME = :file:`C:\\Program Files\\Java\\jdk-11.0.1`
+
+#. Verify mvn is using the correct Java version by running:
+
+   .. code-block:: sh
+
+       mvn -version
+
+   It should report a Java version of 11.0.1.
+
+.. Note::
+
+   In this tutorial we refer to file and directory paths as used by Windows. If you are fortunate
+   enough to be using another operating system such as Linux or OSX all of the commands and source
+   code below will work, just modify the paths to suit. 
+
+Updating the POM
+----------------
+
+A few changes are required to compile and run on Java 11:
+
+#. Add dependencies for **gt-render** and **gt-main**, since we import packages from them:
+
+    .. literalinclude:: artifacts/pom-jdk11.xml
+        :language: xml
+        :start-after: </properties>
+        :end-before: <repositories>
+
+#. Update the Maven plugin versions. We want to ensure we use the most recent, Java 11-compatible, versions. We will also update the configuration of the Maven compile plugin to target Java 11:
+
+    .. literalinclude:: artifacts/pom-jdk11.xml
+        :language: xml
+        :start-after: </repositories>
+        :end-before: </project>
+
+Adding Module Info
+------------------
+
+Java 9 introduced the concept of modules to the JVM. 
+
+A module is a named, self-contained bundle of Java code. Modules may depend upon other modules, and may export packages from themselves to other modules that depend upon them. Module configuration is controled by the :file:`module-info.java` file.
+
+For a more detailed overview of the module system, refer to the `State of the Module System <http://openjdk.java.net/projects/jigsaw/spec/sotms/>`_.
+
+#. Add a :file:`module-info.java` file under :file:`tutorial\src\main\java`. 
+
+#. Name our module ``org.geotools.tutorial.quickstart``:
+
+   .. code-block:: java
+
+        module org.geotools.tutorial.quickstart { }
+
+#. Then, add the modules we depend upon:
+
+   .. literalinclude:: artifacts/module-info.java
+      :language: java
+
+   You'll notice the four geotools modules match those added as dependencies in the :file:`pom.xml`. We also include the ``java.desktop`` module, which contains user interface components required for the app to function.
+
+Running the application
+-----------------------
+        
+#. If you need some shapefiles to work with you will find a selection of data at the
+   http://www.naturalearthdata.com/ project which is supported by the North American Cartographic
+   Information Society. Head to the link below and download some cultural vectors. You can use the 'Download all 50m cultural themes' at top.
+   
+   * `1:50m Cultural Vectors <http://www.naturalearthdata.com/downloads/50m-cultural-vectors/>`_
+   
+   Unzip the above data into a location you can find easily such as the desktop.
+
+#. You can run the application using Maven on the command line::
+   
+     mvn exec:java -Dexec.mainClass=org.geotools.tutorial.quickstart.Quickstart
```

Comment:
That is it? What about module path fun and games?

* https://www.mojohaus.org/exec-maven-plugin/examples/example-exec-for-java-programs.html

Reply:
Maven does it for you as far as I can tell. Some combination of the release version config targeting 11 and declaring the module info results in a state where you don't need to declare anything explicitly.

I tried it out, and it worked fine.